pubsub/rocketmq/rocketmq.go

```diff
@@ -176,7 +182,7 @@ func (r *rocketMQ) Publish(req *pubsub.PublishRequest) error {
 				}
 			}
 
-			ctx, cancel := context.WithTimeout(r.ctx, time.Duration(r.metadata.SendTimeOut))
+			ctx, cancel := context.WithTimeout(r.ctx, time.Duration(r.metadata.SendTimeOut)*time.Second)
```

Comment:
This is another breaking change. In 1.8 we are saying that the value of "sendTimeOut" is in nanoseconds: https://v1-8.docs.dapr.io/reference/components-reference/supported-pubsub/setup-rocketmq/

I do think that using nanoseconds is really bad however

Reply:
@berndverst thoughts?

Reply:
How about keeping `sendTimeOut` as nanosecond, marking it as depecated (ie. removing it from docs) and adding a new option `sendTimeOutSec`?

Reply:
I think this is the better way to deal with it, so I will handle it this way?

Reply:
Sounds good to me! You can also continue accepting `sendTimeOut` (as nanoseconds) and show a warning that the option is deprecated. See an example here (although in this case the option was completely ignored, while in your case you can still accept nanoseconds in `sendTimeOut`): https://github.com/dapr/components-contrib/blob/a08988c25ed32263809d9fdacaac90edb7b005a1/pubsub/azure/servicebus/servicebus.go#L244
 And, thank you 😊